crowdsec-docs/docs/user_guides/replay_mode.md

```diff
@@ -0,0 +1,201 @@
+---
+id: replay_mode
+title: Replay Mode
+sidebar_position: 6
+---
+
+
+While CrowdSec can be used to monitor "live" logs, it can as well be used to replay old log files.
+It is a *great* way to test scenario, detect false positives & false negatives or simply generate reporting on a past time period.
+
+When doing so, CrowdSec will read the logs, extract timestamps from those, so that the scenarios/buckets can be evaluated with the log's timestamps. The resulting overflows will be pushed to the API as any other alert, but the timestamp will be the timestamps of the logs, properly allowing you to view the alerts in their original time line.
+
+
+you can run :
+
+```bash
+sudo crowdsec -c /etc/crowdsec/user.yaml -dsn file:///path/to/your/log/file.log -type log_file_type
+```
+
+Where `-dsn` points to the log file you want to process, and the `-type` is similar to what you would put in your acquisition's label field, for example :
+
+```bash
+sudo crowdsec -c /etc/crowdsec/user.yaml -dsn file:///var/log/nginx/2019.log -type nginx
+sudo crowdsec -c /etc/crowdsec/user.yaml -dsn file:///var/log/sshd-2019.log -type syslog
+sudo crowdsec -c /etc/crowdsec/user.yaml -dns "journalctl://filters=_SYSTEMD_UNIT=ssh.service --since yesterday" -type syslog
```

Comment:
not sure the `--since yesterday` works. Thoughs @blotus ? if it's the case we want to document it in the datasource as well (it's a native journalctl option)